snap/README.md

```diff
@@ -260,11 +248,33 @@ sudo snap set edgexfoundry env.security-proxy.user=user01,USER_ID,ES256
 sudo snap set edgexfoundry env.security-proxy.public-key="$(cat public.pem)"
 ```
 
-The second step is then to generate a token using the user ID which you specified:
+Alternatively, you can create the user using the secrets-config command. You need to provide the following:
```

Comment:
This is not only an alternative, but the only way to create multiple users. As mentioned, the previous commands only allow setting a single user.

I suggest:
~Alternatively, you can create the user using~To create multiple users, use the secrets-config command. You need to provide the following:

Reply:
modified